### ISSUE TITLE
Ubuntu one-click mining program has never seenele token output

### ISSUE THREAD

**Issue by 1994jeff** [seeleteam/go-seele#1066]
I downloaded the one-click mining program linux version on both Ubuntu 14.04 and centos7 servers
Generated the coinbase account address of slice 1, the private key of the p2p node and configured it on the node1.json file
Then use ./node start -c ../node1.json -m stop to synchronize the local block height to more than 130w
Later, I used ./node start -c ../node1.json --threads 50 for mining, but the official account of coinbase has not produced any output. And often there are some error messages, have you encountered such a situation?
It's been several days now, I don't know how to solve it

我在Ubuntu14.04以及centos7的服务器上都下载了一键挖矿程序linux版本
生成了片1的coinbase账号地址，p2p节点私钥并配置在node1.json文件上
然后使用./node start -c ../node1.json -m stop同步本地区块高度至130w以上了
之后使用./node start -c ../node1.json --threads 50进行挖矿，但是官方查看coinbase的账号上一直都没有任何产出。并且时常会有一些错误信息，你们遇到过这样的情况吗？
目前已经好几天了，不知道该如何解决

**Comment by frode88** [comment]
Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.

**Comment by 1994jeff** [comment]
> Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.

Yes, my friend, I use the latest linux1.25 version in the tar file for mining, but there is no reward. Even if I upgrade ubutnu to 16.04, there will still be some errors. I can post the errors after I get home Hope to have a solution.

**Comment by frode88** [comment]
> > Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.
> 
> Yes, my friend, I use the latest linux1.25 version in the tar file for mining, but there is no reward. Even if I upgrade ubutnu to 16.04, there will still be some errors. I can post the errors after I get home Hope to have a solution.

https://medium.com/@SeeleTech/seele-mining-tutorial-simple-version-1806f5b29554

diff high so no guarante for reward. you need good cpu  :)

**Comment by 1994jeff** [comment]
> > > Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.
> > 
> > 
> > Yes, my friend, I use the latest linux1.25 version in the tar file for mining, but there is no reward. Even if I upgrade ubutnu to 16.04, there will still be some errors. I can post the errors after I get home Hope to have a solution.
> 
> https://medium.com/@SeeleTech/seele-mining-tutorial-simple-version-1806f5b29554
> 
> diff high so no guarante for reward. you need good cpu :)

Ok,Thank you . May be the reason. My old computer, the CPU is NVIDIA GT 640, the CPU memory size is 2GB, and the total memory size of the computer is 16GB. Is this configuration impossible to mine? Is it convenient to ask what configuration of the CPU you are using now? How many seele coins can you have in a day with your configuration?

**Comment by 1994jeff** [comment]
> > > Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.
> > 
> > 
> > Yes, my friend, I use the latest linux1.25 version in the tar file for mining, but there is no reward. Even if I upgrade ubutnu to 16.04, there will still be some errors. I can post the errors after I get home Hope to have a solution.
> 
> https://medium.com/@SeeleTech/seele-mining-tutorial-simple-version-1806f5b29554
> 
> diff high so no guarante for reward. you need good cpu :)

If possible I will update my computer configuration to support mining

**Comment by frode88** [comment]
> > > > Just download from this link https://github.com/seeleteam/go-seele/releases and run the miner. no need for one click. btw one click need to be updated manuly anyway.
> > > 
> > > 
> > > Yes, my friend, I use the latest linux1.25 version in the tar file for mining, but there is no reward. Even if I upgrade ubutnu to 16.04, there will still be some errors. I can post the errors after I get home Hope to have a solution.
> > 
> > 
> > https://medium.com/@SeeleTech/seele-mining-tutorial-simple-version-1806f5b29554
> > diff high so no guarante for reward. you need good cpu :)
> 
> Ok,Thank you . May be the reason. My old computer, the CPU is NVIDIA GT 640, the CPU memory size is 2GB, and the total memory size of the computer is 16GB. Is this configuration impossible to mine? Is it convenient to ask what configuration of the CPU you are using now? How many seele coins can you have in a day with your configuration?

I use i7 16 threads
